weblab/experiments/processing.py

```diff
@@ -56,12 +57,18 @@ def submit_experiment(model, model_version, protocol, protocol_version, user, re
 
     # Check there isn't an existing version if we're not allowed to re-run
     if not rerun_ok:
-        version, created = ExperimentVersion.objects.get_or_create(
-            experiment=experiment,
-            author=user,
-        )
+        try:
+            version, created = ExperimentVersion.objects.get_or_create(
+                experiment=experiment,
+                defaults={
+                    'author': user,
+                }
+            )
+        except MultipleObjectsReturned:
+            print('Multi objects')
```

Comment:
Are we printing rather than logging in the front-end?

Reply:
Oops, that was debug output left in by mistake!